components/prize_pool/wikis/commons/starcraft_starcraft2/prize_pool_starcraft.lua

```diff
@@ -0,0 +1,357 @@
+	Variables.varDefine('prize pool table id', prizePoolIndex)
+	if prizePoolIndex == 1 and Logic.readBool(Logic.emptyOr(args.storeTournament, Namespace.isMain())) then
+		CustomPrizePool._storeIntoTournamentLpdb()
+	end
+
+	if Logic.readBool(args.storelpdb) then
+		pageVars:set('placementRecords.' .. prizePoolIndex, Json.stringify(_lpdb_stash))
+	end
+
+	return buildPrizePool
+end
+
+function CustomLpdbInjector:adjust(lpdbData, placement, opponent)
+	local seriesNumber = tonumber(Variables.varDefault('tournament_series_number'))
+	local lastStatuses = {
+		CustomPrizePool._getStatusFromScore(lpdbData.lastscore),
+		CustomPrizePool._getStatusFromScore(lpdbData.lastvsscore),
+	}
+	local extradata = {
+		featured = Variables.varDefault('featured') or 'false',
+		lastStatuses = lastStatuses,
+		placeRange = {placement.placeStart, placement.placeEnd},
+		playernumber = StarcraftOpponent.partySize(opponent.opponentData),
+		points = placement:getPrizeRewardForOpponent(opponent, PRIZE_TYPE_POINTS .. 1),
+		points2 = placement:getPrizeRewardForOpponent(opponent, PRIZE_TYPE_POINTS .. 2),
+		seriesnumber = seriesNumber and string.format('%05d', seriesNumber) or '',
+		wofrom = lastStatuses[1] == WALKOVER_VS_STATUS or nil,
+		woto = lastStatuses[2] == WALKOVER_VS_STATUS or nil,
+	}
+
+	-- make these available for the stash further down
+	lpdbData.liquipediatier = _tier
+	lpdbData.liquipediatiertype = Variables.varDefault('tournament_liquipediatiertype')
+	lpdbData.type = Variables.varDefault('tournament_type')
+
+	lpdbData.weight = Weight.calc(
+		lpdbData.individualprizemoney or 0,
+		lpdbData.liquipediatier,
+		lpdbData.placement,
+		lpdbData.liquipediatiertype,
+		lpdbData.type
+	)
+
+	if type(lpdbData.players) == 'table' then
+		lpdbData.players = StarcraftOpponent.toLpdbStruct(opponent.opponentData).opponentplayers
+	end
+
+	if lpdbData.lastvs then
+		local lastVs = opponent.additionalData.LASTVS
+		extradata.vsOpponent = Table.deepCopy(lastVs)
+		if lastVs.type == StarcraftOpponent.team then
+			lpdbData.lastvs = Json.stringify{
+				type = lastVs.opponenttype,
+				name = lastVs.name
+			}
+		else
+			lastVs = StarcraftOpponent.toLpdbStruct(lastVs) or {}
+			lpdbData.lastvs = Json.stringify(Table.merge(
+					lastVs.opponentplayers or {},
+					{type = lastVs.opponenttype}
+				))
+		end
+	end
+
+	lpdbData.extradata = Table.mergeInto(lpdbData.extradata, extradata)
+
+	lpdbData.mode = CustomPrizePool._getMode(opponent.opponenttype, opponent.opponentData)
+	lpdbData.tournament = _tournament_name
+	lpdbData.series = _series
+
+	local prizePoolIndex = tonumber(Variables.varDefault('prizepool_index')) or 0
+	if prizePoolIndex == 1 and _tournament_extradata_cache[lpdbData.placement or ''] then
+		table.insert(_tournament_extradata_cache[lpdbData.placement], Table.deepCopy(lpdbData))
+	end
+
+	lpdbData.objectName = CustomPrizePool._overwriteObjectName(lpdbData, prizePoolIndex)
+	lpdbData.id = lpdbData.objectName -- wanted for the stash
```

Comment:
TeamCard that uses `.id`?

Reply:
changed sc/sc2 teamcards to read `record.objectName or record.id` so the `lpdbData.id = lpdbData.objectName` line can be kicked